app/src/main/java/com/github/libretube/Subscriptions.kt

```diff
@@ -66,6 +66,12 @@ class Subscriptions : Fragment() {
                 fetchFeed(feedRecView, progressBar, view)
             }
 
+            var toggleSubs = view.findViewById<RelativeLayout>(R.id.toggle_subs)
+            toggleSubs.setOnClickListener {
+                channelRecView.visibility = if (channelRecView.visibility == View.GONE) View.VISIBLE else View.GONE
+                feedRecView.visibility = if (feedRecView.visibility == View.GONE) View.VISIBLE else View.GONE
+            }
```

Comment:
Maybe we can lazy load the subscriptions api request here?

Reply:
Changed it. But the performance shouldn't have changed with my pr, idk